### DISCUSSION TITLE
See all queries/number of all queries made

### DISCUSSION THREAD

**Discussion by haruki-py** [AdguardTeam/AdGuardHome#7200] [Q&A] [answered]
Is there any way to see all the queries made to adguardhome in all time (not 24 hours)? I wanna see how much AGH allowed and blocked

**Comment by Dynamic5912** [comment, marked as answer]
> Is there any way to see all the queries made to adguardhome in all time (not 24 hours)? I wanna see how much AGH allowed and blocked

Increase the stats logging?
